### PR TITLE
1349296781 - AADSUser, I can see the updated modal with three buttons option

### PR DESCRIPTION
**Pull request by frank-shenton**
Before submitting my PR to Code Review, I have made sure:

- [x] I have run the tests using `yarn run test`
- [x] I have exported any new files in index.js
- [x] My file supports Formik if will potentially be used in a form
- [x] The PR name follows the convention `(#<ticket_number>) - <ticket_user_story>`
- [x] I have incremented the version in `package.json` if I want to publish a new version
- [x] If my change is breaking I have updated the minor version number(i.e. the middle one) and don't merge until I have the PR on Corazon ready

### CODE REVIEW

**Comment by frank-shenton** [comment]
image

File: src/assets/cross-icon.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<svg
```
Comment (frank-shenton) [inline src/assets/cross-icon.svg:1]: Replacing the rubbish fontawesome icon!

File: src/components/molecules/ConfirmationModal/ConfirmationModal.tsx [hunk 2]
```diff
@@ -12,8 +13,8 @@ interface IWithTitleProps extends React.HTMLProps<HTMLDivElement> {
 export const HeaderBar = styled.div<IWithTitleProps>`
   width: 100%;
   height: ${({ withTitle }) => (withTitle ? "3.5rem" : "2.875rem")};
-  border-bottom: 0.0625rem solid
-    ${({ withTitle, theme }) => (withTitle ? theme.primary.main.hex : "transparent")};
+  border-bottom: 1px solid
```
Comment (RaquelCasadoCRC) [inline src/components/molecules/ConfirmationModal/ConfirmationModal.tsx:16]: Is there a reason to change this to px?

Comment (frank-shenton) [reply src/components/molecules/ConfirmationModal/ConfirmationModal.tsx:16]: Aha good challenge. I inspected the css of the line on figma and it gave me 1px.
Just double checked and 0.0625rem is the same as 1px with our rem of 16px.

So yeah this change wasn't needed in retrospect !

**Review by RaquelCasadoCRC — APPROVED** [review]

**Review by RaquelCasadoCRC — APPROVED** [review]

**Pull request merged**